connector.yaml

```diff
@@ -2,7 +2,25 @@ version: "1.0"
 specification:
   name: dynamodb
   summary: A DynamoDB source plugin for Conduit
-  description: A DynamoDB source plugin for Conduit, it scans the table at the beginning taking a snapshot, then starts listening to CDC events using DynamoDB streams.
+  description: |
+    ## Source
+    A source connector that pulls data from a DynamoDB table to downstream resources via Conduit.
+
+    The connector starts with a snapshot of the data currently existent in the table, sends these records to the
+    destination, then starts the CDC (Change Data Capture) mode which will listen to events happening on the table
+    in real-time, and sends these events' records to the destination (these events include: `updates`, `deletes`, and `inserts`).
+    You can opt out from taking the snapshot by setting the parameter `skipSnapshot` to `true`, meaning that only the CDC
+    events will be captured.
+
+    The source connector uses [DynamoDB Streams](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html) to get CDC events,
+    so you need to enable the stream before running the connector. Check out the documentation for [how to enable a DynamoDB Stream](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html#Streams.Enabling).
+
+    ## Destination
+    A destination connector that takes a conduit record and stores it into a DynamoDB table.
```

Comment:
We should add some instructions on how to use the destination, for instance, you need to create the table in DynamoDB, because the connector doesn't do it, right?

Reply:
done